service/cache-service/src/test/java/uk/gov/gchq/palisade/cache/service/impl/MockCacheService.java

```diff
@@ -40,14 +42,18 @@ public static void setMock(final CacheService mock) {
         MockCacheService.mock = mock;
     }
 
-
     @Override
-    public CompletableFuture<Boolean> add(final AddCacheRequest request) {
+    public <V> CompletableFuture<Boolean> add(AddCacheRequest<V> request) {
```

Comment:
Absolutely it can. The cache service provides a way to store objects from different services and even one service may be storing different object types.

Reply:
👍